be/src/http/action/update_config_action.cpp

```diff
@@ -142,6 +143,14 @@ Status UpdateConfigAction::update_config(const std::string& name, const std::str
             (void)StorageEngine::instance()->update_manager()->get_pindex_thread_pool()->update_max_threads(
                     max_thread_cnt);
         });
+        _config_callback.emplace("number_tablet_writer_threads", [&]() {
+            const int MIN_TABLET_WRITER_COUNT = 8;
```

Comment:
It's better to place the common variable in the commom.h

Reply:
Done